src/jit/lsra.cpp

```diff
@@ -182,14 +182,21 @@ unsigned LinearScan::getWeight(RefPosition* refPos)
             GenTreeLclVarCommon* lclCommon = treeNode->AsLclVarCommon();
             LclVarDsc*           varDsc    = &(compiler->lvaTable[lclCommon->gtLclNum]);
             weight                         = varDsc->lvRefCntWtd;
+            if (refPos->getInterval()->isSpilled)
+            {
+                // Decrease the weight if the interval has already been spilled.
+                weight -= BB_UNITY_WEIGHT;
```

Comment:
I think that we will need to make this a function call.  (maybe fgGetUnityWeight() )
While BB_UNITY_WEIGHT is correct for methods that do not contain profile data, I think that we want the fgCalledCount value here when  we have profile data.

Reply:
@briansull - I believe that using `fgCalledCount` here (and elsewhere) is incorrect. The `getBBWeight()` method normalizes the block weights to BB_UNITY_WEIGHT, even in the presence of profile data. I think I should restore the changes to use `BB_UNITY_WEIGHT`.
Note that changing this to use `fgCalledCount` resulted in regressions in the frameworks, where the profile data is available.

Reply:
OK, If you have a JitDump of one of these cases I could take a look